submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/security/web/PermissionResource.kt

```diff
@@ -0,0 +1,22 @@
+package ac.uk.ebi.biostd.security.web
+
+import ac.uk.ebi.biostd.security.domain.service.PermissionService
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@PreAuthorize("hasAuthority('ADMIN')")
+class PermissionResource(private val permissionsService: PermissionService) {
+
+    @PutMapping("/permissions")
```

Comment:
This could be part of the `SecurityResource` IMO

Reply:
same as in https://github.com/EBIBioStudies/biostudies-backend-services/pull/404#discussion_r660781216